wscript

```diff
@@ -66,14 +66,20 @@ def build(bld):
         )
 
     bld(features="subst",
-        source= "org.gnome.hamster.service.in",
-        target= "org.gnome.hamster.service",
+        source= "org.gnome.Hamster.service.in",
+        target= "org.gnome.Hamster.service",
```

Comment:
If you don't mind, I'd like to be sure I understand the difference between the `org.gnome.Hamster` and `org.gnome.Hamster.GtkApp` (currently called `org.gnome.Hamster.Windows`) services.

Because what they do might have an impact on the naming to follow if we want Hamster to work properly inside Flatpak. (which I'd very much want)

Reply:
`org.gnome.Hamster` is the database server:
https://github.com/projecthamster/hamster/blob/72a8e318b0d9095f4d0eae45d76ba7b06122d7af/src/hamster-service.py#L46

`org.gnome.Hamster.WindowServer` is the legacy application windows server:
https://github.com/projecthamster/hamster/blob/72a8e318b0d9095f4d0eae45d76ba7b06122d7af/src/hamster-windows-service.py#L24

Reply:
Be aware that this "little" change with "hamster" becoming "Hamster" can cause quite some headaches for users already having those files.
If users start complaining about services not starting any more for strange reasons, they need to perform a "sudo rm /usr/share/dbus-1/services/org.gnome.hamster.*" to remove the old files

Reply:
*Thanks for the warning, 
but beware that postings on already merged issues are hard to remember.*
So dbus launch is not case sensitive ?
Anyway there have been several other issues with old install remnants.
Warning set on the release page, in the wiki Troubleshooting section,
and updated README (ab5e9ea43efb4f8f571db7df6620d2d0db144cf5).

Reply:
>  So dbus launch is not case sensitive ?

It is.

It's just that if people installed Hamster manually (i.e not with a package) then the old service files will be kept on the system, in parallel to the old ones, when those users updated their Hamster installation by simply running `waf install` from a source tree.

Given that you also renamed the desktop files, the old desktop files would also linger on the system in the same way.

That has the potential to cause lots of confusion.

Personally, I'm of the opinion that people who compile/install their software manually from sources, without going through the safer packaging routes, are expected to be capable of dealing with this kind of problems, and the notice you added to the wiki/readme seems perfectly sufficient to me.

Side note:

> because to comply with freedesktop specifications some files are camel-cased now.

The spec mandates a few things like the dotted-notation, the absence of hyphens (`-`) and that each component starts with a letter. It does not mandate capitalizing the first letter of the last component. This is merely the emerging convention, but lots of apps use lower-case and that doesn't cause any problem and never should. I assume you did that on my suggestion, so I apologize for not having made that sufficiently clear.

Reply:
> ... the notice you added to the wiki/readme seems perfectly sufficient to me.

Thanks for the discussion, it's reassuring.

> I assume you did that on my suggestion

Partly, but your suggestions were perfectly inline with both
the freedesktop recommendations and the flatpak conventions.
Let's get to the bottom of it in #547. 
If we can, it would be much better to make all file changes in one go.